BibBuddy/app/src/main/java/de/bibbuddy/NoteDao.java

```diff
@@ -274,7 +274,12 @@ public String findTextById(Long id) {
    * @return returns the notes text value without formatting texts
    */
   public String findStrippedTextById(Long id) {
-    return findTextById(id).replaceAll("<.*?>", "");
+    return findTextById(id).replaceAll(
+        "(<p dir=\"ltr\"( style=\"margin-top:0; margin-bottom:0;\")?>|</p>|"
+            + "<div align=\"right\"  >|<div align=\"center\"  >|</div>|"
+            + "<span style=\"text-decoration:line-through;\">|</span>|<(/)?i>|"
+            + "<(/)?b>|<(/)?u>|<(/)?br>|<(/)?blockquote>)",
+        "");
```

Comment:
The error correction is at the wrong place. You have to escape all the special characters like "<>" or "&" when saving the text.

Reply:
As far as I remember (and just read in the protocol), we decided to not do that. The reason is that we need the formatting strings for proper display in the app - only when we export, we want to strip these. That's also the way it was implemented before :)

Reply:
Ok, then we leave it like this and do not change it.